.editorconfig

```diff
@@ -0,0 +1,24 @@
+# WARNING - THIS FILE WAS GENERATED BY THE dps-gradle-spring-boot GRADLE PLUGIN
```

Comment:
doesn't need to be included in this project, can be added to .gitignore unless you're going to amend it?

Reply:
OK.  Just added it because that's what the plugin did... I'll add to .gitignore and remove...